src/orquestra/sdk/_base/_driver/_ce_runtime.py

```diff
@@ -32,6 +37,10 @@
 
 from . import _client, _exceptions, _models
 
+CE_REQUIRES_PYTHON_VERSION: Version = Version(
+    original="3.9", major=3, minor=9, patch=0, is_prerelease=False
+)
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏CE actually uses 3.9.17. How will we make sure this is updated in the future?